cmd/parse.go

```diff
@@ -66,7 +69,12 @@ func (self *parsingContext) Parse() {
 				if count > *number_of_records {
 					return
 				}
-				serialized, _ := json.MarshalIndent(event, " ", " ")
+				if *parse_json_compact {
+					serialized, _ = json.Marshal(event)
```

Comment:
This is not exactly jsonl - if writing to a file wouldnt this be all wrong?

would be be a better command liine interface to the have a flag with output format like --format jsonl 
then have different encoding routines. Maybe similar to

https://github.com/Velocidex/velociraptor/blob/ee414779bd0418931f4ad26743e97af3978f9c0c/bin/query.go#L60